alienfile

```diff
@@ -44,29 +47,52 @@ share {
   # -e test_checksums
   # -e checksum
 
+  my $CC = qq(CC=@{[ $Config::Config{cc} ]});
+  my $static_lib = 'libdeflate.a';
+  my $shared_lib = qq(libdeflate.@{[ $Config::Config{so} ]});
+  my @progs = map { "$_@{[ $Config::Config{_exe} ]}" } qw(gzip gunzip);
   build [
-    '%{gmake} libdeflate.a gzip gunzip checksum CFLAGS=-fPIC',
-    '%{gmake} libdeflate.so',
+    [ '%{gmake}',
+        $CC,
```

Comment:
I guess this could be `CC=%{cc}` but since Config is loaded already.

Reply:
Whichever you prefer! I can change it if you would like.